index.html

```diff
@@ -0,0 +1,300 @@
+                                <ul class="list-view">
+                                    <li class="list-item">Ruby on rails</li>
+                                    <li class="list-item">css</li>
+                                    <li class="list-item">JavaScript</li>
+                                    <li class="list-item">html</li>
+                                </ul>
+                            </div>
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div class="card-body">
+                            <h5>
+                                Multi-Post Stories<br>
+                                Gain+Glory
+                            </h5>
+                            <div class="work-card-list">
+                                <ul class="list-view">
+                                    <li class="list-item">Ruby on rails</li>
+                                    <li class="list-item">css</li>
+                                    <li class="list-item">JavaScript</li>
+                                    <li class="list-item">html</li>
+                                </ul>
+                            </div>
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div class="card-body">
+                            <h5>
+                                Multi-Post Stories<br>
+                                Gain+Glory
+                            </h5>
+                            <div class="work-card-list">
+                                <ul class="list-view">
+                                    <li class="list-item">Ruby on rails</li>
+                                    <li class="list-item">css</li>
+                                    <li class="list-item">JavaScript</li>
+                                    <li class="list-item">html</li>
+                                </ul>
+                            </div>
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            <!-- About me section -->
+            <div class="about-me main-l-item">
+                <section class="about-me-describe">
+                    <h3>
+                        About me
+                    </h3>
+
+                    <div class="about-description">
+                        I can help you build a product,
+                        feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded,
+                        don’t hestiate to contact me.
+                    </div>
```

Comment:
Please ensure that the paragraph (e.g., "I can help you build a product ...") is a `p` tag according to the project requirement.

Reply:
Fixed